### ISSUE TITLE
Shaman stuck trying to leave Dalaran (Flight Point/Mount)[Can't do that while moving bug]

### ISSUE THREAD

**Issue by JenyTulwartz** [dalaranwow/dalaran-wow#5157]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: On my Shaman, when i travel from Dalaran, albeit on mount or on flight path, I get stuck in mid air. Right now I took a flight path from Dalaran to K3, I am floating a little bit above the ground and when I move, I run in place. I can not re-mount, can't really do anything to get out of it, it just states "Can't do that while moving". When I do this on my mount, I get a little bit away from Dalaran and just stop mid air, I can hop off my mount but i'm just stuck. The only way that I could fix this is just an ALT+F4 and relog into the game (I can't even 'log out/quit game' while stuck like this either).

**Expected behaviour**: Let me get out of Dalaran or mount off the flight path like my other characters.

**Steps to reproduce the problem**:

1. Leave Dalaran on my Shaman
2. Flight path = Gets to destination but can't move | Mount = Flies for a bit, then gets stuck once a bit out of Dalaran
3. Give up and force quit relog

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Thank you for the report @JenyTulwartz.
Issue probably related to #2878
